doc/en/conf.py

```diff
@@ -397,7 +398,7 @@
 }
 
 
-def configure_logging(app: "sphinx.application.Sphinx") -> None:
```

Comment:
Did pyupgrade did this, or you updated it manually?

Reply:
Pyupgrade removes stringification on type declarations when adding the annotations future

Reply:
Noice